app/controllers/users_controller.rb

```diff
@@ -1,4 +1,20 @@
 class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :verify, :update]
+
   def show
   end
+
+  def verify
+    user = User.find(params[:user_id])
+    user.update_attribute(:verified, true)
+    redirect_to profile_path
+    # maybe add this line ?
+    # flash[:success] = "Thank you! Your account has been activated."
+  end
+
+private
+
+  def set_user
+    @user = current_user
+  end
```

Comment:
I thought about putting a set_user in the teams_controller, but I wasn't sure how beneficial it would be. Do we prefer to use set_user so we can call @user instead of current_user in the methods?

Reply:
I think we eventually use a facade, so maybe we don't need this right now. Or we can just handle the cleanup when we refactor to use facades later